pkg/generator_test.go

```diff
@@ -517,6 +517,10 @@ func (s *GeneratorSuite) TestGeneratorForEmptyInterface() {
 	s.checkGeneration("empty_interface.go", "Blank", false, "", "")
 }
 
+func (s *GeneratorSuite) TestGeneratorForPanicOnNoReturnValueInterface() {
+	s.checkGeneration("panic_err.go", "PanicOnNoReturnValue", false, "", "")
+}
+
```

Comment:
This is not a useful style of test, I'm trying to move away from these. Just create a test that asserts the behavior of the mock object.

Thanks for tackling this!

Reply:
Hey @LandonTClipp! Thanks for suggesting this! 

Yeah, I just created the test for the new message when the test panics and deleted this one. Thanks for the heads up! I thought this was something necessary as well.